### ISSUE TITLE
Bug: PIXI.VideoResource will not be updated automatically

### ISSUE THREAD

**Issue by lzy100p** [pixijs/pixijs#9727]
### Current Behavior

If the video is not played successfully at the first time and then clicked to play it again, PIXI.VideoResource will not be updated automatically.Even if the video has played properly (you can hear the video sound)

### Expected Behavior

Play video normally。PIXI.VideoResource can be automatically updated when the video is played.

### Steps to Reproduce

example：
```js
import * as PIXI from 'pixi.js';

const app = new PIXI.Application({ backgroundAlpha: 0, resizeTo: window });

document.body.appendChild(app.view);

// Create play button that can be used to trigger the video
const button = new PIXI.Graphics()
    .beginFill(0x0, 0.5)
    .drawRoundedRect(0, 0, 100, 100, 10)
    .endFill()
    .beginFill(0xffffff)
    .moveTo(36, 30)
    .lineTo(36, 70)
    .lineTo(70, 50);

// Position the button
button.x = (app.screen.width - button.width) / 2;
button.y = (app.screen.height - button.height) / 2;

// Enable interactivity on the button
button.eventMode = 'static';
button.cursor = 'pointer';

// Add to the stage
app.stage.addChild(button);

const texture = PIXI.Texture.from('https://pixijs.com/assets/video.mp4');

// create a new Sprite using the video texture (yes it's that easy)
const videoSprite = new PIXI.Sprite(texture);
app.stage.addChild(videoSprite);
button.on('pointertap', onPlayVideo);
function onPlayVideo()
{
    // Don't need the button anymore
    button.destroy();

    // create a video texture from a path

    texture.baseTexture.resource.source.play()
    // Stetch the fullscreen
    videoSprite.width = app.screen.width;
    videoSprite.height = app.screen.height;
}
```

### Environment

- **`pixi.js` version**: *e.g. 7.3.1*
- **Browser & Version**: *e.g. Chrome 108*
- **OS & Version**: *e.g. Ubuntu 22.04*
- **Running Example**: *e.g. <https://pixiplayground.com/>*

**Comment by dev7355608** [comment]
I assume this would be fixed by #9707.

I wasn't able to reproduce it with the code above in pixiplayground.com + Chrome. With a small addition of `source.currentTime = 1` right before the `source.play()` is fails though: https://pixiplayground.com/#/edit/HZ_L0Dd-HBjEbjraAqgyf

With the fix of #9707 it works: https://pixiplayground.com/#/edit/KihvDkM5h6OZ3QB_aGEVx

**Comment by Zyie** [comment]
Looks like this was fixed, feel free to reopen if you still have this issue
